changelog/fix_fix_operatormethodcall_when_forwarding.md

```diff
@@ -0,0 +1 @@
+* [#11378](https://github.com/rubocop/rubocop/pull/11378): Fix OperatorMethodCall when forwarding arguments. ([@sambostock][])
```

Comment:
Can you tweak it?

```suggestion
* [#11377](https://github.com/rubocop/rubocop/issues/11377): Fix `Style/OperatorMethodCall` when forwarding arguments. ([@sambostock][])
```

Reply:
Oh, haha, real life race condition. I guess the issue was opened while I was implementing the fix. I'll update!